.NET/Microsoft.Recognizers.Definitions.Common/English/DateTimeDefinitions.cs

```diff
@@ -255,7 +256,7 @@ public static class DateTimeDefinitions
       public static readonly string RelativeDurationUnitRegex = $@"(?:(?:(?<=({NextPrefixRegex}|{PreviousPrefixRegex}|{ThisPrefixRegex})\s+)({DurationUnitRegex}))|((the|my))\s+({RestrictedTimeUnitRegex}))";
       public static readonly string ReferenceDatePeriodRegex = $@"\b{ReferencePrefixRegex}\s+(?<duration>week|month|year|decade|weekend)\b";
       public const string ConnectorRegex = @"^(-|,|for|t|around|@)$";
-      public const string FromToRegex = @"\b(from).+(to)\b.+";
+      public const string FromToRegex = @"(\b(from).+(to)\b.+|\b(from)\s+\d{4}\s+and\s+\d{4})";
```

Comment:
I don't get this regex. Why is this needed?
It seems this is more of a workaround to the issue? Then it seems to be very brittle and will fail for cases like:
"from january and february" or for "from 2002 or 2003".

Reply:
If this regex is just kind of a filter to avoid other cases falling into the ExperimentalMode behaviour, then we should rename it.

Reply:
it was meant to exclude only the cases like "from 2002 and 2003". Now I modified it to 
(\b(from).+(to|and|or)\b.+)
but maybe this is too unrestricted?